app/views/spree/components/auth/_change_form.html.erb

```diff
@@ -0,0 +1,40 @@
+<%
+  html = { class: "auth-form" }
+  url = spree.update_password_path
+%>
+
+<%= form_for @spree_user, as: :spree_user, url: url, method: :put, html: html do |f| %>
+  <div class="auth-form__input-wrapper">
```

Comment:
Why is this wrapper needed?

Reply:
To decouple auth form input positioning from the input component itself.